omod/src/main/java/org/openmrs/module/oauth2login/web/Utils.java

```diff
@@ -21,4 +32,31 @@ public static File getFileInAppDataDirectory(String filename) {
 		return Paths.get(OpenmrsUtil.getApplicationDataDirectory(), filename).toFile();
 	}
 	
+	public static String getPostLogoutRedirectUrl(HttpServletRequest request) throws IOException {
+		Properties properties = PropertyUtils.getProperties(PropertyUtils.getOAuth2PropertiesPath());
+		String redirectPath = properties.getProperty("logoutUri");
+		//the redirect path can contain a [token] that should be replaced by the auth token
+		if (StringUtils.isNoneBlank(redirectPath) && redirectPath.contains("[token]")) {
```

Comment:
Since you are dealing with just once string, is there a reason why you did not use the obvious isNotBlank?